GameData/Kerbalism/Profiles/Default.cfg

```diff
@@ -127,13 +127,30 @@ Profile
     relax_message = $ON_VESSEL$KERBAL is breathing again
   }
 
+  Rule
+  {
+    name = humidity
+    input = MoistAtmosphere
```

Comment:
What does it mean to have the same input and output? Does this actually produce MoistAtmosphere?

Reply:
At the very least this is worthy of a comment so the poor souls who read this can make sense of it ;-)

Reply:
Comments added 😉